src/Models/ForeignRelationship.php

```diff
@@ -7,9 +7,9 @@
 use CrestApps\CodeGenerator\Support\Helpers;
 use CrestApps\CodeGenerator\Support\ResourceMapper;
 use CrestApps\CodeGenerator\Support\Str;
-use DB;
+use Illuminate\Support\Facades\DB;
```

Comment:
Now testbench is hooked up and we can reference Laravel facades, um.. reference them.